.travis.yml

```diff
@@ -1,27 +1,30 @@
 language: node_js
-matrix:
-  fast_finish: true
 node_js:
-- '6'
-branches:
-  except:
-  - build
+  - '8'
 env:
   global:
-  - CXX=g++-4.8
-addons:
-  apt:
-    sources:
-    - ubuntu-toolchain-r-test
-    - deadsnakes
-    packages:
-    - g++-4.8
-    - python3.5
+    # kosssi GITHUB_TOKEN
+    - secure: "Y4NN0e37W6OplWw+M0WLjIkh1yrtzNNqqWwJcKh+RRKK7ZHiShjWwdHQyzSsbxUjifh08d9BjEnbCZ6lHAfFhGbBxIWWZyyr4xqF/b4lMh+0WWXgFP03ESB9WBRFcUBso7QC4kgDtads+CZrG/0TWgruKma6iNFCV87Gbr3jznz+3HmILiYdgZtHwF4FWl4u/R7DbqTXmtHJT7WYDEeWUKUvDVdDPH+q3uzWGSlU0XU2hLf+AsnAagDpnyMD7v2XSgZRrIWPl0TMdiIGIK+/gyk6T1XBJVhx6YIoPxcB984N9oBdigMbzxLWb7zPeIicW425McbiZ0bYrDia60tFpc2EC6GDYReYGBuNnHVqAoYT0+hExlfn8zn1QITa788p7jbaJUZBD1vzg2eJG+gZM1gnfeUBoYLikEtmY3U2jgDjaWUSLX+Cwy/StZagUOG5BGg1qgxBeaR1Za1aDClHncSEcO//CjWHkd9aTxS2N61Y6IrfftrKnj92SDniSNFGsvtxrMp6dPjIqPRpfVYUJ8hTcfwMLDgUETGpafqFeFj2solVknCgPIJBMmI48syc0kUWZaalpkTNtwgui+7Ax6ERcs0U10F6mVN/xlAaL7Bw/TmQjYtgFOna36g3Fd99PY77YbDDdsuKmMTdHPSswQfgXHQ9Qiy0LzeEYnwzzyQ="
 cache:
   yarn: true
   directories:
-  - node_modules
+    - node_modules
 script:
-- yarn build
-after_success:
-  - test $TRAVIS_BRANCH = "master" && yarn run deploy:travis -- --branch=build
+  - yarn build
+deploy:
+  - provider: pages
```

Comment:
Do we need to have a site for every konnector ? I guess the README would be sufficient ?

Reply:
Understood, pages is a "hack" to git push -f on the branch provided